src/common/transformations/include/transformations/utils/utils.hpp

```diff
@@ -196,6 +196,10 @@ TRANSFORMATIONS_API void visit_shape_path(ov::Node* node,
                                           std::unordered_set<ov::Node*>& visited,
                                           std::function<void(ov::Node*)> func);
 
+TRANSFORMATIONS_API void visit_constant_path(ov::Node* node,
```

Comment:
Consider add doxygen to explain what is doing on constant path?

Reply:
Added, thanks